package-lock.json

```diff
@@ -1,9925 +1,8 @@
 {
   "name": "node-red-contrib-kolibri",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Not sure why this file changed this much

Reply:
you need to update your npm to version >7